Python/compile.c

```diff
@@ -7951,6 +7951,24 @@ assemble(struct compiler *c, int addNone)
     return co;
 }
 
+static PyObject*
+get_const_value(int opcode, int oparg, PyObject *co_consts)
+{
+    PyObject *constant = NULL;
+    assert(HAS_CONST(opcode));
+    if (opcode == LOAD_CONST) {
+        constant = PyList_GET_ITEM(co_consts, oparg);
+    }
+
+    if (constant == NULL) {
+        PyErr_SetString(PyExc_SystemError,
+                        "Internal error: failed to get value of a constant");
+        return NULL;
+    }
```

Comment:
This seems like a good case to use switch-case, especially when it gets complicated through `LOAD_NONE`, `LOAD_COMMON` etc. (with perhaps an inner switch for the latter that is depending on the oparg).

Reply:
Sure, we can turn it to a switch once we have more than one case.